norfair/camera_motion.py

```diff
@@ -151,7 +153,11 @@ def abs_to_rel(self, points: np.ndarray):
         points_transformed = points_transformed / points_transformed[:, -1].reshape(
             -1, 1
         )
-        return points_transformed[:, :2]
+        new_points_transformed = points_transformed[:, :2]
+        if np.isnan(new_points_transformed).any() or np.isinf(new_points_transformed).any():
+            new_points_transformed = np.array([[0, 0], [0, 0]])
+
+        return new_points_transformed
```

Comment:
Great catch. I assume this is happening when dividing by the last column of the `points_transformed` array, since it might contain zero values (really weird case, but sure, can happen). Tell me if you think the source of the problem might be other, because I actually couldn't reproduce the nan or infs error.

Anyway, I'm not exactly convinced that this is the best way of fixing it if you get infinite values. The two main reasons I say that are
1. You are just returning an array of two points, but take into account that the number of points that this function is receiving can be arbitrary. The expected shape of the output array should be the same as the input.
2. The 'arbitrarily' chosen 0 coordinates for the output also doesn't sit well with me. We could instead just add some really small value to the last column of the `points_transformed` array whenever it has some zero entry, so that when we compute the division we are not getting a division by zero. I think at least that feels somewhat less arbitrary than just returning the zero point. 

Could you please try the suggestion of my second point and check if you still can reproduce the nan/inf error or if that would also solve it?